nat/src/stateless/config/prefixtrie.rs

```diff
@@ -202,8 +181,8 @@ mod tests {
             .expect("Failed to insert prefix");
 
         pt.insert_ipv6(
-            prefix_v6("aa:bb:cc:dd::/32"),
-            "prefix_aa:bb:cc:dd::/32".to_string(),
+            prefix_v6("aa:bb:cc:dd::/64"),
+            "prefix_aa:bb:cc:dd::/64".to_string(),
```

Comment:
Why does this need to change? It's probably a concern if inserting the `/32` doesn't work anymore?

Reply:
Ah you mean it's a requirement now. OK, makes sense to adjust the test then, thank you.